packages/phone-number-privacy/combiner/src/config.ts

```diff
@@ -88,6 +96,10 @@ if (DEV_MODE) {
       threshold: functionConfig.threshold_signature.threshold_signature_threshold,
       polynomial: functionConfig.threshold_signature.threshold_polynomial,
     },
+    keyVersions: {
+      phoneNumberPrivacy: functionConfig.keyVersions.phoneNumberPrivacy || 1,
+      domains: functionConfig.keyVersions.domains || 1,
```

Comment:
Using `??` instead of `||` is more ideomatic.
```suggestion
      phoneNumberPrivacy: functionConfig.keyVersions.phoneNumberPrivacy ?? 1,
      domains: functionConfig.keyVersions.domains ?? 1,
```